oam/install_scripts/post-install

```diff
@@ -159,16 +173,30 @@ mkdir -p $installdir/data/bulk/rollback >/dev/null 2>&1
 mkdir -p $installdir/data/bulk/tmpjob >/dev/null 2>&1
 rm -f $installdir/data/bulk/tmpjob/* >/dev/null 2>&1
 
-#create columnstore temp file directory
-mkdir -p /tmp/columnstore_tmp_files >/dev/null 2>&1
+#get temp base directory
+tmpDir="/tmp"
+if [ $user != "root" ]; then
+	tmpDir=$HOME"/.tmp"
+	mkdir $tmpDir >/dev/null 2>&1
+else
+	#get columnstore temp file directory name
+	TempFileDir=`$installdir/bin/getConfig SystemConfig TempFileDir`
+	tmpDir=${tmpDir}${TempFileDir}
+	mkdir $tmpDir >/dev/null 2>&1
+fi
+
+#set base columnstore temp file directory
+$installdir/bin/setConfig -d SystemConfig SystemTempFileDir $tmpDir
```

Comment:
Can we consolidate some of these config entries?  Looks like this tmp dir is at SystemConfig/TempFileDir and SystemConfig/SystemTempFileDir...?  Is that intentional?